boards/aithinker/ai_m61_32s_kit/ai_m61_32s.dtsi

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
```

Comment:
I think the version is usually in the `.dts` and it's already in `ai_m61_32s_kit.dts`.

That isn't making any difference though, just for avoiding possible confusion.